.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+Refs #
+
+- [ ] Add documentation.
+- [ ] Add tests.
+- [ ] Add a changelog entry.
+- [ ] Add your name in the contributors file.
+- [ ] Update the [API_VERSION](https://github.com/Kinto/kinto/blob/master/kinto/__init__.py#L15) if you changed the http API and [in the doc](https://github.com/Kinto/kinto/blob/master/docs/api/index.rst)
```

Comment:
nit: If you changed the HTTP API, update the API_VERSION constant and add an API changelog entry in the docs